### PR TITLE
Vivek/bankloadtest

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

**Comment by tamird** [comment]
LGTM modulo nits

File: examples/bank/bank.go [hunk 1]
```diff
@@ -33,16 +33,26 @@ import (
...
+// These two flags configure a range of accounts over which the program functions.
+var firstAccount = flag.Int("first-account", 0, "First account in the account range.")
+var numAccounts = flag.Int("num-accounts", 1000, "Number of accounts in the account range.")
+
+var numParallelTransfers = flag.Int("num-parallel-transfers", 100, "Number of parallel transfers.")
+
 // Makes an id string from an id int.
-func makeAccountID(id int) []byte {
-	return []byte(fmt.Sprintf("%09d", id))
+func (bank *Bank) makeAccountID(id int) []byte {
```
Comment (petermattis) [inline examples/bank/bank.go]: You should move this method below where the `Bank` struct is defined. Feels odd to see it first.

File: examples/bank/bank.go [hunk 2]
```diff
@@ -188,15 +216,20 @@ func main() {
...
-	// Run a test cockroach instance to represent the bank.
-	security.SetReadFileFn(securitytest.Asset)
-	serv := server.StartTestServer(nil)
-	defer serv.Stop()
 	// Initialize the bank.
 	var bank Bank
-	bank.numAccounts = 10
+	bank.firstAccount = *firstAccount
+	bank.numAccounts = *numAccounts
 	// Create a database handle
```
Comment (petermattis) [inline examples/bank/bank.go]: Should move this comment down to where the database handle is created.

File: examples/bank/bank.go [hunk 3]
```diff
@@ -188,15 +216,20 @@ func main() {
...
-	db, err := client.Open("https://root@" + serv.ServingAddr() + "?certs=test_certs")
+	name := *dbName
+	if *dbName == "" {
+		// Run a test cockroach instance to represent the bank.
+		security.SetReadFileFn(securitytest.Asset)
+		serv := server.StartTestServer(nil)
+		defer serv.Stop()
+		name = "https://root@" + serv.ServingAddr() + "?certs=test_certs"
```
Comment (petermattis) [inline examples/bank/bank.go]: You could also assign to `*dbName` here and get rid of the `name` variable.

**Comment by petermattis** [comment]
LGTM

**Pull request merged**